lighthouse-core/audits/seo/font-size.js

```diff
@@ -34,17 +39,17 @@ function getUniqueFailingRules(fontSizeArtifact) {
     }
   });
 
-  return failingRules.valuesArray();
+  return [...failingRules.values()];
```

Comment:
I'd have an extraordinarily small preference for `Array.from`, mostly commenting out of curiosity if this is your preference? :)

Reply:
hmm, I used to prefer `Array.from` since it felt more deliberate and less like an action in place or something, but now I have spread blindness and see them as the same thing, so I don't think I care anymore...